chapter_13_database_layer_validation.asciidoc

```diff
@@ -573,6 +599,19 @@ The current situation is that we have one view and URL for displaying a list,
 and one view and URL for processing additions to that list.  We're going to
 combine them into one. So, in 'list.html', our form will have a different
 target:
+// CSANAD: Since we are working in a not completely working state, I would
+// already mention or move the NOTE here from the end of the subsection:
+//   "Refactor: Transferring the new_item Functionality into view_list".
+// And then just re-reference it from where it is right now.
+//
+// I think adding a few words on how this is not just a simple refactor but it's
+// going to get us to our working state would help the reader recognise these
+// situations later, independently.
+// Something like pointing out how the FT is partially passing for:
+//  - adding one empty list item results in an error message
+//  - adding one valid list item is also passing
+//  - the error occurs when we try adding an empty list item again
+// from which we conclude why we are refactoring the views first
```

Comment:
i think calling this out is a good idea yes.  i think maybe the rigorous thing to do might be to add an early `return` to the FT, with a `# TODO` on it, so that it still runs right up to the known error, but we have green fts, then we do a commit, then we refactor, then we commit, then we undo the early return... something like that?  would you like to have a go at it?